### ISSUE TITLE
Root folder Project64.rdb bug.

### ISSUE THREAD

**Issue by Frank-74** [project64/project64#313]
Just discovered this.

Found that a blank line is needed at the top of the file, otherwise settings are not applied.  Also without a blank line, you get duplicated settings.

Only happens on the first game in Project64.rdb for 1.6 plugins.

**Comment by project64** [comment]
this is with the 1.6 plugins ?

**Comment by Frank-74** [comment]
Yes, only with 1.6 plugins.  First game in list is not read without a blank line at the top of the file.

**Comment by project64** [comment]
do we actually need a rdb file in the root directory, since we do not actually have the 1.6 plugins for 2.2?

**Comment by Frank-74** [comment]
Lots of games need 1.6.1 instead of 1.7.  40 Winks has black textures in 1.7, Monaco Grand Prix you can't see the car setup options in 1.7,  Majora's Mask LOT and day transition, Oot subscreen fix, Perfect Dark scanners and video interference effect in 1.6.1.  Also Winback still needs 1.5.2 to get rid of the floating grey square.  1.6 plugins is the only way to set the Clear Frame=2 option needed for Perfect Dark.
